### PR TITLE
removing duplicated key in .clang-format

### PR DESCRIPTION
**Pull request by GitPaean**
not saying it is used or useful, but my IDE complained about it.

### CODE REVIEW

**Comment by GitPaean** [comment]
Please suggest.  We have the same thing in other repos also.

**Comment by GitPaean** [comment]
jenkins build this please

File: .clang-format [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
         BasedOnStyle: WebKit,
-        AlignAfterOpenBracket: AlwaysBreak,
         AlignConsecutiveAssignments: false,
         AlignConsecutiveDeclarations: false,
         AlignAfterOpenBracket: Align,
```
Comment (GitPaean) [inline .clang-format:5]: it is repeated here with different values.

**Review by akva2 — APPROVED** [review]
Obviously fine and I agree with the one chosen, but not just my call to make.

**Comment by atgeirr** [comment]
> not just my call to make

I agree that Align is best, so let's just merge!

**Pull request merged**